package.json

```diff
@@ -14,10 +14,11 @@
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
     "prepare:translations": "sh scripts/prepare-translations.sh",
-    "link-typedoc-plugin": "ln -s $(pwd)/typedoc-plugin-audius-theme $(pwd)/node_modules/typedoc-plugin-audius-theme"
+    "link-typedoc-plugin": "ln -s $(pwd)/typedoc-plugin-audius-theme $(pwd)/node_modules/typedoc-plugin-audius-theme",
+    "init-sdk-docgen": "cd $(pwd)/node_modules/@audius/sdk && npm i && cd ../../../ && npm run link-typedoc-plugin"
```

Comment:
:o

Reply:
Yeah I wanted to do something more elegant like `npm i @audius/sdk --also=dev` but that didn't seem to work